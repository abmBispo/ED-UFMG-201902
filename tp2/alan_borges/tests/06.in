2000 10 3
993 hzh
614 ial
35 kyz
387 mxw
533 ffw
381 sqa
698 lfc
703 jwn
839 nbs
118 cdn