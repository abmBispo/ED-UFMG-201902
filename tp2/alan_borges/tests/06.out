1 cdn 118
1 ffw 533
1 kyz 35
1 mxw 387
1 sqa 381
2 ial 614
2 lfc 698
3 jwn 703
3 nbs 839
4 hzh 993
